.github/tests/besu-all.yaml

```
@@ -1,14 +1,15 @@
 participants:
-  - el_type: geth
-    cl_type: teku
-  - el_type: geth
-    cl_type: teku
   - el_type: besu
     cl_type: prysm
+  - el_type: besu
+    cl_type: teku
   - el_type: besu
     cl_type: nimbus
   - el_type: besu
     cl_type: lighthouse
   - el_type: besu
     cl_type: lodestar
-additional_services: []
+  - el_type: besu
+    cl_type: grandine
+additional_services:
+  - tx_spammer
```

```suggestion
additional_services: []
```

These tests are ran overnight. There is no need to test tx_spammer here. You can test that locally.